app/models/parser/base.rb

```diff
@@ -166,13 +157,7 @@ def get_twitter_metadata
 
     def twitter_author_url(username)
       return if bad_username?(username)
-      begin
-        twitter_client.user(username)&.url&.to_s
-      rescue Twitter::Error => e
-        PenderSentry.notify(e, url: url, username: username)
-        Rails.logger.warn level: 'WARN', message: "[Parser] #{e.message}", username: username, error_class: e.class
-        nil
-      end
+        "https://twitter.com/" + username[1..]
```

Comment:
Minor change suggestion: Remove the leading space
Question: `username[1..]` is because the first character is always `@`? If so, can we please add a comment just to make it super clear?

Reply:
**suggestion** I'd actually like to make a different suggestion to make the code a bit clearer to me - I'd suggest using gsub (i.e. username.gsub("@",""). that supports both a case where username is prefixed with @ and isn't, and is more in line with what we do in other classes